### PR TITLE
Mode:add Plane AUTOLAND mode

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: mavutil.py [hunk 1]
```diff
@@ -2148,6 +2148,7 @@ def mode_string_v09(msg):
     23 : 'QACRO',
     24 : 'THERMAL',
     25 : 'LOITERALTQLAND',
+    26 : 'AUTLOAND',
```
Comment (peterbarker) [inline mavutil.py:2151]:
```suggestion
    26 : 'AUTOLAND',
```

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: mavutil.py (hunk 1 above)
Comment (Hwurzburg) [reply mavutil.py]: duh

File: mavutil.py [hunk 2]
```diff
@@ -2148,6 +2148,7 @@ def mode_string_v09(msg):
     23 : 'QACRO',
     24 : 'THERMAL',
     25 : 'LOITERALTQLAND',
+    26 : 'AUT0LAND',
```
Comment (peterbarker) [inline mavutil.py:2151]:
```suggestion
    26 : 'AUTOLAND',
```

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: mavutil.py (hunk 2 above)
Comment (Hwurzburg) [reply mavutil.py]: double duh

**Review by peterbarker — APPROVED** [review]

**Pull request merged**